python/py-awscli/Portfile

```diff
@@ -30,15 +29,12 @@ if {${name} ne ${subport}} {
     depends_build-append \
                         port:py${python.version}-setuptools
     depends_lib-append \
-                        port:py${python.version}-asn1 \
-                        port:py${python.version}-bcdoc \
                         port:py${python.version}-botocore \
                         port:py${python.version}-colorama \
-                        port:py${python.version}-dateutil \
                         port:py${python.version}-docutils \
-                        port:py${python.version}-jmespath \
                         port:py${python.version}-rsa \
-                        port:py${python.version}-six
```

Comment:
Did you remove these dependencies because botocore includes them transitively? Or does AWS CLI actually not require them anymore?